packages/create-react-native-library/templates/example/example/windows/{%- project.name %}Example/Package.appxmanifest

```diff
@@ -0,0 +1,50 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  IgnorableNamespaces="uap mp">
+
+  <Identity
+    Name="cb1b1ded-2f74-4bf4-89cd-ab411a2b6aa9"
+    Publisher="CN=Bartosz Klonowski"
```

Comment:
this should probably be the user's name or something in the template :)

Reply:
I completely forgot about this, thanks!
I will double check whether there are more of such mistakes.